app/policies/application_policy.rb

```diff
@@ -71,8 +71,8 @@ def support_admin?
     user.has_role?(:support_admin)
   end
 
-  def user_is_banned?
-    user.banned
+  def user_is_suspended?
```

Comment:
Nitpick: generally it's a bit unidiomatic to use `is` in predicate methods in Ruby, `user_suspended?` would be the more common form. But that would lead to a lot of extra changes, so is probably not worth it.

```suggestion
  def user_suspended?
```

Reply:
Unfortunately, we sorta do this everywhere :laughing: I'll fix this instance in this PR and follow up with a 15-minute fix for a few other places that I've seen it!